cairoplot/tests.py

```diff
@@ -1,6 +1,6 @@
 import cairo, math, random
 
-import cairoplot
+from . import cairoplot
```

Comment:
cairoplot is embedded source from another project, so it may be interesting to compare against porting work by others, such as https://github.com/m-labs/cairoplot3

Reply:
Fixed in fdff3f9.